webpack.config.test.js

```diff
@@ -48,15 +40,16 @@ module.exports = {
         watchFiles: ['src/**/*', 'examples/src/**/*'],
     },
     plugins: [
-        new RemoveEmptyScriptsPlugin({}),
-        new MiniCssExtractPlugin({
-            filename: '[name].css',
-        }),
-        new HtmlWebpackPlugin({
-            template: 'examples/src/index.html',
-            filename: 'index.html',
-            inject: false,
-            minify: false,
-        }),
+        new HtmlBundlerPlugin({
+            entry: {
+              index: 'examples/src/index.html',
```

Comment:
The syntax is identically to Webpack `entry`.
The output file is `examples/dist/index.html`.
The key of the entry option is an output file without `.html` extension.